[Fix] Refactor text alignment keyboard shortcuts

benniekiss opened the pull request:
<!--
  Thank you for opening up this pull request! Please read this comment carefully
  and make sure to fill in as much info as possible below as well.

  First, the obligatory checklist. You must make sure to follow this list as
  much as possible to make merging your PR as easy as possible:

  1. I documented all behaviour within the code as far as I could.
  2. This PR targets the develop branch and *not* the master branch.
  3. I have tested my changes extensively and paid extra attention to potential
     cross-platform issues (e./g. Cmd/Ctrl/Super key bindings)
  4. I ran the `yarn lint` and `yarn test` commands successfully
  5. I have added an entry to the CHANGELOG.md
  6. I matched my code-style to the repository as far as possible
  7. I agree that my code will be published under the GNUP GPL v3 license
  8. In case I created new files, I added a header to them (see the existing
     files for examples)
  9. Before opening this PR, I ran `git pull` a last time to prevent any merge
     issues

  If you don't know how to fulfill some of the above points, feel free to open
  your PR nevertheless and mention those points where you are unsure. The more
  you can fulfill, the faster we can merge your PR, otherwise it will just take
  longer.
 -->

## Description
<!-- Below, please describe what the PR does in one or two short sentences. -->
This PR refactors the text alignment keyboard shortcuts so that they no longer conflict, notably with `Ctrl-C` copy and `Ctrl-L` insert ID.

Closes #5860

## Changes
<!-- What changes did you make? Please explicitly state any breaking API changes so that nobody is confused why other components suddenly stop working -->
A `Shift` key was added to each command so that they do not conflict with other main editor commands. 

Center alignment was changed to `Ctrl-Shift-E`, which matches LibreOffice and MS Word (`Ctrl-e`). `Ctrl-Shift-L` and `Ctrl-Shift-R` already match `Ctrl-L` and `Ctrl-R`, respectively.

macOS commands were added.

## Additional information
<!-- If there is anything else that might be of interest, please provide it here -->

<!-- Please provide any testing system -->
Tested on: <!-- OS including version --> macOS 15.6

kyaso reviewed (commented):
Thank you for the fix, looks good to me! Good point with matching the bindings with MS Word / Libre. Again, from my side everything is fine, I'll let @nathanlesage have the last word, once he's back!

benniekiss commented:
Good catch! I would suggest `Alt`, but that conflicts with the dark mode toggle, which could also be changed.

benniekiss commented:
I decided to change both Copy Link ID and Toggle dark mode. I wanted to keep the `Ctrl-Shift-*` mod for the alignment commands since it is more ergonomic to me, so I moved Copy Link ID to `Ctrl-Alt-L`, and I moved Toggle Dark Mode to `Ctrl-Alt-D`.

benniekiss commented:
`Cmd-Shift-D` works for dark mode, if we can't get around the system shortcut conflict.

nathanlesage commented:
Ah damn, that's a good point, and I actually use Cmd-Alt-D quite often 😵‍💫

This is what I hate about shortcuts, there are just too many of them 🫠

My take at this moment would actually be: just remove the alignment shortcuts entirely. We can readd that via the custom shortcuts PR later on. Like, for real, if we will always have conflicts, just keep the existing ones and remove those alignment ones for now ...

benniekiss commented:
Sounds good to me! I'll update this to just drop the alignment commands. Would prefer me comment them out, but leave them in the code; or delete them entirely?

nathanlesage commented:
Almost: I'd rather not have block comments but individual `//` line comments (with one additional line comment atop of those with a `TODO` keyword). That should make the diff more readable.

Also, the import can be completely removed; the linter will remind us that we have to re-import that command in the future :)
